apps/ui/src/apis/rides/request-to-join-ride.ts

```diff
@@ -0,0 +1,64 @@
+import { graphqlRequest } from '../graphql';
+import { DEFAULT_LAT_LONG } from '../utils/defaultConst';
+
+interface PostRequestToJoinRideResponse {
+  createUserRide: {
+    userRide: {
+      userId: number;
+      rideId: number;
+      status: string;
+      fromLat: number;
+      fromLong: number;
+    };
+  };
+}
+const getUpdateQuery = (
+  userId: number,
+  rideId: number,
+  status: string,
+  toLat?: number,
+  toLong?: number,
+  fromLat?: number,
+  fromLong?: number,
```

Comment:
why are the start and destination locations optional?

Reply:
Because it is currently not in the UI and I wanted to check and also because it is flexible in terms of pickup location